From 58a4f8e6463885e2ce5a20a76266fc5676dfff9e Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Fri, 18 Jan 2013 15:46:52 +0200
Subject: [PATCH 085/184] bone-common: ti,vcc-aux-disable-is-sleep enable

Enable vcc-aux-disable-is-sleep on both MMC interfaces.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 arch/arm/boot/dts/am335x-bone-common.dtsi | 1 +
 firmware/capes/cape-bone-2g-emmc1.dts     | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-bone-common.dtsi b/arch/arm/boot/dts/am335x-bone-common.dtsi
index 4787d8c..96bce02 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -359,6 +359,7 @@
 &mmc1 {
 	status = "okay";
 	vmmc-supply = <&ldo3_reg>;
+	ti,vcc-aux-disable-is-sleep;
 };
 
 &edma {
diff --git a/firmware/capes/cape-bone-2g-emmc1.dts b/firmware/capes/cape-bone-2g-emmc1.dts
index c81a986..e2a4a46 100644
--- a/firmware/capes/cape-bone-2g-emmc1.dts
+++ b/firmware/capes/cape-bone-2g-emmc1.dts
@@ -48,6 +48,7 @@
 			ti,non-removable;
 			status = "okay";
 			reset-gpios = <&gpio2 20 1>;	/* active low */
+			ti,vcc-aux-disable-is-sleep;
 		};
 	};
 };
-- 
1.9.0

